src/ng/directive/input.js

```diff
@@ -1308,7 +1308,7 @@ function baseInputType(scope, element, attr, ctrl, $sniffer, $browser) {
 
     // if user modifies input value using context menu in IE, we need "paste" and "cut" events to catch it
```

Comment:
This comment needs updating now :smiley:

Reply:
Thank's.